packages/renderer/src/lib/onboarding/Onboarding.svelte

```diff
@@ -34,6 +34,11 @@ let onboardings: OnboardingInfo[];
 let activeStep: ActiveOnboardingStep;
 $: activeStep;
 
+// Content may be dynamically updated via "when" statements
+// this allows us to show warnings when clicking buttons, etc
+let activeStepContent: OnboardingStepItem[][];
+$: activeStepContent;
```

Comment:
not sure what this line does @cdrage

Reply:
it makes the array reactive to refresh the page, no?

Reply:
Yup, is this correct svelte convention?

Reply:
Yup, is this correct svelte convention? @benoitf

Reply:
AFAIK reactivity is done on the assignment in the subscribe method

Reply:
Today I learned! https://sustainablewww.org/principles/should-you-use-sveltes-store-subscribe-or-the-reactivity-dollar-sign

@lstocchi 

Referenced in subscribe = no need for $